Added CICO JMeter script for Orchid release

OlKondratenko opened the pull request.

mtraneis reviewed (approved):
Thanks for checking in many scripts here.
1. Please delete "_release" from the name. Calling it "Nolana" or "Fameflower" is good enough. We all know what it means.
2. I believe we agreed to delete loans/jmeter-supported-data folder and its credentials file. 
3. What happens to the existing CICO JMeter script? Is it being renamed and moved into one of these dir's?  We want to preserve it as it has a lot of history.

OlKondratenko commented:
> Thanks for checking in many scripts here.
> 
> 1. Please delete "_release" from the name. Calling it "Nolana" or "Fameflower" is good enough. We all know what it means.
> 2. I believe we agreed to delete loans/jmeter-supported-data folder and its credentials file.
> 3. What happens to the existing CICO JMeter script? Is it being renamed and moved into one of these dir's?  We want to preserve it as it has a lot of history.

1. Done
2. Yes, we agreed to delete loans/jmeter-supported-data
3. I replaced existing CICO JMeter script to \check-in-check-out\Honeysuckle folder according to the release name.
